src/fastoad_cs25/models/loads/sizing_loads/sizing_loads_envelope.py

```diff
@@ -0,0 +1,119 @@
+        self.add_input("data:weight:aircraft:MTOW", val=np.nan, units="kg")
+        self.add_input("data:mission:sizing:cs25:gust:load_factor_1", units="unitless")
+        self.add_input("data:mission:sizing:cs25:gust:load_factor_2", units="unitless")
+        self.add_input("data:mission:sizing:cs25:maneuver:load_factor_1", units="unitless")
+        self.add_input("data:mission:sizing:cs25:maneuver:load_factor_2", units="unitless")
+
+        self.add_output("data:mission:sizing:cs25:envelope:max_load_factor_1", units="unitless")
+        self.add_output("data:mission:sizing:cs25:envelope:max_load_factor_2", units="unitless")
+        self.add_output("data:mission:sizing:cs25:envelope:max_sizing_load_1", units="N")
+        self.add_output("data:mission:sizing:cs25:envelope:max_sizing_load_2", units="N")
+
+    def setup_partials(self):
+        # Load factor outputs
+        self.declare_partials(
+            "data:mission:sizing:cs25:envelope:max_load_factor_1",
+            [
+                "data:mission:sizing:cs25:gust:load_factor_1",
+                "data:mission:sizing:cs25:maneuver:load_factor_1",
+            ],
+            method="fd",
+        )
+        self.declare_partials(
+            "data:mission:sizing:cs25:envelope:max_load_factor_2",
+            [
+                "data:mission:sizing:cs25:gust:load_factor_2",
+                "data:mission:sizing:cs25:maneuver:load_factor_2",
+            ],
+            method="fd",
+        )
+
+        # Sizing load outputs
+        self.declare_partials(
+            "data:mission:sizing:cs25:envelope:max_sizing_load_1", "*", method="fd"
+        )
+        self.declare_partials(
+            "data:mission:sizing:cs25:envelope:max_sizing_load_2", "*", method="fd"
+        )
+
+    def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
+        mzfw = inputs["data:weight:aircraft:MZFW"]
+        mfw = inputs["data:weight:aircraft:MFW"]
+        mtow = inputs["data:weight:aircraft:MTOW"]
+
+        n1_maneuver = inputs["data:mission:sizing:cs25:maneuver:load_factor_1"]
+        n2_maneuver = inputs["data:mission:sizing:cs25:maneuver:load_factor_2"]
+        n1_gust = inputs["data:mission:sizing:cs25:gust:load_factor_1"]
+        n2_gust = inputs["data:mission:sizing:cs25:gust:load_factor_2"]
+
+        # load case #1
+        m1 = 1.05 * mzfw
+        n1m1_maneuver = n1_maneuver * m1 * g
+        n1m1_gust = n1_gust * m1 * g
+
+        # load case #2
+        if not self.options["fuel_load_alleviation"]:
+            n2m2_maneuver = n2_maneuver * mtow * g
+            n2m2_gust = n2_gust * mtow * g
+        else:
+            mcv = min(0.8 * mfw, mtow - mzfw)  # fuel mass in the overhanging part of wing
+            n2m2_maneuver = n2_maneuver * (mtow - 0.55 * mcv) * g
+            n2m2_gust = n2_gust * (mtow - 0.55 * mcv) * g
+
+        n1 = max(n1_gust, n1_maneuver)
+        n2 = max(n2_gust, n2_maneuver)
+        n1m1 = max(n1m1_gust, n1m1_maneuver)
```

Comment:
Isn't that simply:
```suggestion
        n1m1 = n1 * m1 * g
```
?
And same for n2